<launch>
  <node pkg="rapid_pbd" type="pr2_action_node" name="pbd_pr2_action_node" />
  <node pkg="rapid_pbd" type="program_execution_node" name="pbd_program_executor_node">
    <remap from="rapid_pbd/l_arm_joint_action" to="l_arm_controller/follow_joint_trajectory" />
    <remap from="rapid_pbd/r_arm_joint_action" to="r_arm_controller/follow_joint_trajectory" />
  </node>
</launch>
